src/components/Countries.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { NavLink, useParams } from 'react-router-dom';
+import style from './Countries.module.scss';
+import { getCountries } from '../redux/Countries';
+import numberWithCommas from '../functions/numberWithCommas';
+import sumFieldOfArray from '../functions/sumFieldOfArray';
+import defaultState from '../redux/CountriesDefaultState';
+
+function Countries() {
+  // const continents = useSelector((state) => state.countriesReducer.continents);
+  const dispatch = useDispatch();
+  const { continent } = useParams();
+  const [countries, setCountries] = useState([]);
+  const selectedCountries = defaultState.countries.filter((el) => el.continent === continent).map((el) => el.name).join(',');
+
+  useEffect(() => {
+    fetch(`https://corona.lmao.ninja/v2/countries/${selectedCountries}`)
+      .then((response) => response.json())
+      .then((result) => {
+        setCountries(result.map((el) => ({
+          name: el.country,
+          flag: el.countryInfo.flag,
+          todayCases: el.todayCases,
+          todayDeaths: el.todayDeaths,
+        })));
+        dispatch(getCountries(countries));
+      }).catch((error) => `Error: ${error}`);
+  }, []);
+
```

Comment:
I like your implementation of getting the flag 👏🏻

Reply:
Thanks